source/code/include/playfab/PlayFabPlatformMacros.h

```diff
@@ -29,4 +29,24 @@
 // Hence to specify only Windows, we have check for ! _DURANGO.
 #if defined(_WIN32) && !defined(_DURANGO)
 #define PLAYFAB_PLATFORM_WINDOWS
-#endif //_WIN32
\ No newline at end of file
+#endif //_WIN32
+
+#if defined(PLAYFAB_PLATFORM_WINDOWS) || defined(PLAYFAB_PLATFORM_XBOX)
+typedef signed __int64 Int64;
+typedef signed __int32 Int32;
```

Comment:
in a previous version I mentioned how redefining the fixed-width numeric types (e.g. int32_t) provided by the std library is unnecesary.

You mentioned you'd be willing to improve it in a separate change. Can you post the link to the work item for that implied punted work?

Reply:
If you have a suggestion for a better overall solution here, please let me know.
For now I'm just copy-pasting what's already there, into an appropriate file.